next.config.js

```diff
@@ -83,7 +83,7 @@ const securityHeaders = [
 module.exports = {
   webpack: (config, { isServer }) => {
     config.externals = [...config.externals, 'pino-pretty'];
-    if (isServer) {
+    if (isServer && process.env.NODE_ENV === 'production') {
```

Comment:
This removes the warning by nextjs which reverted the config changes anyway in dev.

```
Warning: Reverting webpack devtool to 'false'.
Changing the webpack devtool in development mode will cause severe performance regressions.
Read more: https://nextjs.org/docs/messages/improper-devtool
```